src/buffer/buffer.rs

```diff
@@ -385,15 +379,15 @@ impl Buffer {
     }
 }
 
-impl Debug for Buffer {
+impl std::fmt::Debug for Buffer {
     /// Writes a debug representation of the buffer to the given formatter.
     ///
     /// The format is like a pretty printed struct, with the following fields:
     /// * `area`: displayed as `Rect { x: 1, y: 2, width: 3, height: 4 }`
     /// * `content`: displayed as a list of strings representing the content of the buffer
     /// * `styles`: displayed as a list of: `{ x: 1, y: 2, fg: Color::Red, bg: Color::Blue,
     ///   modifier: Modifier::BOLD }` only showing a value when there is a change in style.
-    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
```

Comment:
I've seen this change in other PRs, but was wasn't sure what the purpose of it was. Can you clarify this?
I'd generally by practice try to avoid writing a bare Result like this to avoid confusion, so I can understand it there, but `Debug` and `Formatter` seem benign enough.

Reply:
Mainly: get rid of the `use Result` which at best confuses and at worst introduces subtle bugs in other parts of the code without noticing. I also like to see what exactly is implemented for the local struct and as its most often only one time it's imported to be used once. So less clear to read and adds import which is used only once.

Reply:
Sounds fine.

Reply:
Did this more uniformly in #1087